.travis.yml

```diff
@@ -29,9 +29,11 @@ jobs:
         - make defconfig
         - scripts/kconfig/merge_config.sh .config kconfig/base-defconfig kconfig/sof-defconfig
         - KCFLAGS="-Wall -Werror" make -j`getconf _NPROCESSORS_ONLN`
+        - make bindeb-pkg -j`getconf _NPROCESSORS_ONLN`
```

Comment:
wondering if you can leave the regular make command and do the bindeb-pkg once the kernel actually builds. The Debian stuff can be slow, you'd want to compile first without it.

Reply:
@plbossart 
```
        - KCFLAGS="-Wall -Werror" make -j`getconf _NPROCESSORS_ONLN`
        - make bindeb-pkg -j`getconf _NPROCESSORS_ONLN`
```

The first line is the original make. Then followed the deb build

Reply:
Ah yes, sorry. that's fine then